docs/coding-guidelines/css.md

```diff
@@ -275,9 +275,9 @@ selector {
 - DO use a single space before an opening brace
```

Comment:
we should update the line above this about spaces around parenthesis as well

Reply:
I already updated that.